src/core/generator/generator.rs

```diff
@@ -57,7 +57,7 @@ impl Generator {
             query: "Query".into(),
             mutation: None,
             inputs: Vec::new(),
-            type_name_prefix: "T".into(),
+            type_name_prefix: "GEN__".into(),
```

Comment:
Ideally searching for `"GEN__` should only return one result. Move the prefix to a static const.